src/control/cmd/dmg/storage.go

```diff
@@ -19,12 +19,13 @@ import (
 
 // storageCmd is the struct representing the top-level storage subcommand.
 type storageCmd struct {
-	Scan     storageScanCmd     `command:"scan" description:"Scan SCM and NVMe storage attached to remote servers."`
-	Format   storageFormatCmd   `command:"format" description:"Format SCM and NVMe storage attached to remote servers."`
-	Query    storageQueryCmd    `command:"query" description:"Query storage commands, including raw NVMe SSD device health stats and internal blobstore health info."`
-	Set      setFaultyCmd       `command:"set" description:"Manually set the device state."`
-	Replace  storageReplaceCmd  `command:"replace" description:"Replace a storage device that has been hot-removed with a new device."`
-	Identify storageIdentifyCmd `command:"identify" description:"Blink the status LED on a given VMD device for visual SSD identification."`
+	Scan       storageScanCmd     `command:"scan" description:"Scan SCM and NVMe storage attached to remote servers."`
+	Format     storageFormatCmd   `command:"format" description:"Format SCM and NVMe storage attached to remote servers."`
+	Query      storageQueryCmd    `command:"query" description:"Query storage commands, including raw NVMe SSD device health stats and internal blobstore health info."`
+	NvmeRebind nvmeRebindCmd      `command:"nvme-rebind" description:"Unbind NVMe SSD from kernel driver and bind instead to a user-space driver to enable DAOS to use the device."`
```

Comment:
nit: This description is somewhat verbose for a command-line argument. Instead, I recommend something more succinct, such as, "Detach NVMe SSD from kernel driver and rebind to SPDK for use with DAOS."

Reply:
moving into a follow-up PR

Reply:
"rebind to SPDK" isn't correct, do you mean "rebind to VFIO"? I would prefer "rebind to userspace driver"

Reply:
https://github.com/daos-stack/daos/pull/7913

Reply:
Sure, whichever is correct. My goal was primarily to encourage brevity.